src/workers/wallet.ts

```
@@ -817,6 +945,200 @@ export class WalletWorker {
+              (channelState === State.STATE_FLUSHING ||
+                channelState === State.STATE_FLUSHCOMPLETE)
+            ) {
+              return v
+            }
+            break
+          case ChannelState.UPGRADE_ERROR:
+            // if there is no error receipt in the chain state, return v
+            const errorReceipt = await this.chain.rest.ibc.getUpgradeError(
+              v.port_id,
+              v.channel_id
+            )
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)